src/app/dim-ui/Sheet.tsx

```diff
@@ -223,12 +229,12 @@ export default function Sheet({
 
       if (
         dragHandle.current?.contains(e.target as Node) ||
-        sheetContents.current!.scrollTop === 0
+        (enableContentScrollTopDrag && sheetContents.current!.scrollTop === 0)
```

Comment:
Going to call out that I really don't like this behaviour as it isn't intuitive but it's been around forever so we shouldn't change it. I have disabled it in the loadout edit drawer as it gets a bit janky when you are dragging items inside the drawer.